Builds using docker image currently fail: "could not read version from GOROOT"

pulling the tinygo-zoo repo and running:

```sh
$ make build-arduino-colorlamp
```

results in
```sh
docker run --rm -v "/home/max/src/github.com/tinygo/tinygo-zoo:/src" -v "/home/max:/gohost" -e "GOPATH=/home/max:/gohost" tinygo/tinygo tinygo build -o /src/build/arduino-colorlamp.hex -target arduino /src/arduino-colorlamp/main.go
error: could not read version from GOROOT: /usr/local/go
make: *** [Makefile:10: build-arduino-colorlamp] Error 1
```

is GOROOT a required/new environment variable that the image does not provide?

--- a/Dockerfile
+++ b/Dockerfile
@@ -11,7 +11,7 @@ ENV GO_RELEASE=1.12.5
 RUN wget https://dl.google.com/go/go${GO_RELEASE}.linux-amd64.tar.gz && \
     tar xfv go${GO_RELEASE}.linux-amd64.tar.gz -C /usr/local && \
     rm go${GO_RELEASE}.linux-amd64.tar.gz && \
-    find /usr/local/go -mindepth 1 -maxdepth 1 ! -name 'src' -exec rm -rf {} +
+    find /usr/local/go -mindepth 1 -maxdepth 1 ! -name 'src' ! -name 'VERSION' -exec rm -rf {} +
 ENV PATH=${PATH}:/usr/local/go/bin
 
 ENV TINYGO_RELEASE=0.6.0

